src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage.py

```diff
@@ -576,27 +579,26 @@ def _table_acl_scenario(self, table):
         acl = sorted(s.cmd('storage table policy list -t {}'.format(table)).keys())
         assert acl == ['test1', 'test2', 'test3', 'test4']
         s.cmd('storage table policy show -t {} -n test1'.format(table),
-            checks=JMESPathCheck('permission', 'a'))
+              checks=JMESPathCheck('permission', 'a'))
         s.cmd('storage table policy show -t {} -n test2'.format(table),
-            checks=JMESPathCheck('start', '2016-01-01T00:00:00+00:00'))
+              checks=JMESPathCheck('start', '2016-01-01T00:00:00+00:00'))
         s.cmd('storage table policy show -t {} -n test3'.format(table),
-            checks=JMESPathCheck('expiry', '2018-01-01T00:00:00+00:00'))
+              checks=JMESPathCheck('expiry', '2018-01-01T00:00:00+00:00'))
         s.cmd('storage table policy show -t {} -n test4'.format(table), checks=[
             JMESPathCheck('start', '2016-01-01T00:00:00+00:00'),
             JMESPathCheck('expiry', '2016-05-01T00:00:00+00:00'),
             JMESPathCheck('permission', 'raud')
         ])
         s.cmd('storage table policy update -t {} -n test1 --permission au'.format(table))
         s.cmd('storage table policy show -t {} -n test1'.format(table),
-            checks=JMESPathCheck('permission', 'au'))
+              checks=JMESPathCheck('permission', 'au'))
         s.cmd('storage table policy delete -t {} -n test1'.format(table))
         acl = sorted(s.cmd('storage table policy list -t {}'.format(table)).keys())
         assert acl == ['test2', 'test3', 'test4']
 
     def body(self):
         s = self
         table = s.table
-        rg = s.resource_group
```

Comment:
I remove the unused variable. I don't think it will break any logic. Please confirm.